### PR TITLE
Update filename, source URL, and re-render 2.3.1

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: graphviz-{{ version }}.zip
-  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
```
Comment (xflr6) [inline recipe/meta.yaml:11]: This is now actually the preferred URL, see https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package#comment-27606213

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,8 +7,8 @@ package:
...
+  fn: python-graphviz-{{ version }}.zip
```
Comment (xflr6) [inline recipe/meta.yaml:10]: As mentioned in #9, I would rather prefer to keep the official name for the sdist download

Comment (jakirkham) [reply recipe/meta.yaml:10]: Same [response]( https://github.com/conda-forge/python-graphviz-feedstock/pull/9#discussion_r117899997 ).

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:11]: If the URLs work reliably, then we can use `files.pythonhosted.org`. However, for a long time that wasn't the case (even after that linked comment was posted). Hence why `pypi.io` is used throughout conda-forge and recommended in our [example]( https://github.com/conda-forge/staged-recipes/blob/f73e2912d7b7ad5814d11b49e26b6044a3cf8525/recipes/example/meta.yaml#L18 ).

**Pull request merged**